docs/gen-client-docs.py

```diff
@@ -275,6 +296,23 @@ def _to_typed_args(argspec: inspect.FullArgSpec, required: bool) -> list[TypedAr
             out.append(TypedArgument(name=arg, tpe=tpe, default=defaults.get(arg)))
         return out
 
+    def class_properties(self, inst) -> list[PropertyDoc]:
+        property_docs = []
+        for name in dir(inst):
+            if name[0] == '_':
+                # private members
+                continue
+            instance_attr = getattr(inst, name)
+            if name.startswith('_'):
+                continue
+            if inspect.ismethod(instance_attr):
+                continue
+            property_docs.append(
+                PropertyDoc(name=name,
+                            doc=instance_attr.__doc__,
+                            tpe=instance_attr.__class__.__name__ if instance_attr.__class__ != property else None))
```

Comment:
When is the `else` branch taken?

Reply:
Not needed. It was needed with the first version which was not filtering everything correctly.